lib/matplotlib/backend_bases.py

```diff
@@ -3604,21 +3610,28 @@ def _tool_triggered_cbk(self, event):
 
     def _add_tool_cbk(self, event):
         """Captures 'tool_added_event' and adds the tool to the toolbar"""
-        image = self._get_image_filename(event.tool.image)
+        tool, group, position = event.tool, event.data['group'], \
+                                                event.data['position']
+        if group is None:
+            return
```

Comment:
I don't understand... I thought you replaced ``in_toolbar=False`` with ``group=None``...